pulpcore/cli/rpm/common.py

```diff
@@ -1 +1 @@
-CHECKSUM_CHOICES = ("unknown", "md5", "sha1", "sha224", "sha256", "sha384", "sha512")
+CHECKSUM_CHOICES = ("md5", "sha1", "sha224", "sha256", "sha384", "sha512")
```

Comment:
So i think we need this now:
```suggestion
LEGACY_CHECKSUM_CHOICES = ("md5", "sha1", "sha224", "sha256", "sha384", "sha512")
CHECKSUM_CHOICES = ("sha256", "sha384", "sha512")
```